lib/inc/cpp-pcp-client/protocol/serialization.hpp

```diff
@@ -96,7 +96,7 @@ inline void serialize(const T& thing,
     } catch(const std::exception& e) {
         throw message_serialization_error { e.what() };
     } catch(...) {
-        throw message_serialization_error { "seriliazation: unexpected error "
+        throw message_serialization_error { "serialization: unexpected error "
```

Comment:
Ah, I think you misinterpreted that. It was explicitly saying avoid using any Boost.Locale functions directly, because they don't appear to pickup all the locale information across DLL boundaries on Windows. You can use `leatherman::locale::translate` and `leatherman::locale::format` anywhere.

If you want to avoid a dependency on `leatherman/locale/locale.hpp` in the header, you could also add helpers for generating the error message, or a helper for doing the resize and wrapping exceptions.